### ISSUE TITLE
A11y_Model Builder_Create New Experiment_Keyboard: Tool tips that are present for each field are not keyboard accessible. Keyboard focus is not going to tool tips in 'Create New experiment' dialog

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1578]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
#A11y_Model Builder; #A11yAuto; #TabStops; #A11yMAS; #A11yTCS; #WCAG2.1.1; #Keyboard; #Win32; #DesktopApp; #A11ySev2; #ModelBuilder-Win32-Jun2021; #Visual Studio Client;

### Environment Details:
Application name: Model Builder

### OS Details:
Windows Version 21H1 OS Build (19043.1083)
Microsoft Visual Studio

### Note:
This issue is observed in all popups. In 'Create new workspace', 'Create new compute' tool tips are not keyboard accessible

### Repro Steps:

1. Enable ML.NET Model builder as a preview feature in tools>options>Environment>preview features.
2. Create a .NET C# project and tab till project under solution explorer and Add machine learning.
3. Tab till Image classification under scenario tab and press enter.
4. Select Azure under environment tab and tab till set up workspace and press enter.
5. In 'Create New Experiment' dialog, tab till Subscriptions/Machine Learning workspace/compute name fields
6. Verify whether keyboard focus is landing on tool tips present for respective fields.

### Actual Result:
Tool tips that are present for each field are not keyboard accessible. Keyboard focus is not going to tool tips in 'Create New experiment' dialog

### Expected Result:
Tool tips that are present for each field should be keyboard accessible. Keyboard focus should go to tool tips in 'Create New experiment' dialog and info inside tool tip should be visible when keyboard focus lands on tool tip

### User Impact:
Keyboard users will not know the info that is present in tool tips if they are not receiving keyboard focus

**Comment by beccamc** [comment]
Validate on rel/candidate build 16.7.2.2141903. See issue 1627 for instructions for install.

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. Tool tips present for all fields are keyboard accessible. Hence closing the issue.

https://user-images.githubusercontent.com/83570638/131445150-88a84e90-b376-40c2-8d1d-fd41bf8401cc.mp4

**Comment by VijayaBharathyA** [comment]
#Closed;
